src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Arrivals/TransportEquipmentGoodsRefStepDef.scala

```diff
@@ -45,14 +45,14 @@ class TransportEquipmentGoodsRefStepDef extends BaseStepDef {
       .submitPage()
   }
 
-  And("""^(?:I )?click the (.*) link on the You have added (.*) goods item numbers? page""") {
+  And("""^(?:I )?click the (.*) link on the You have added (.*) goods item numbers? page$""") {
     (sectionLink: String, numberOfGoodsItems: String) =>
       AddAnotherGoodsItemNumberPage
         .loadPage(numberOfGoodsItems)
         .clickByPartialLinkText(sectionLink)
   }
 
-  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove goods item number (.*) page""") {
+  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove goods item number (.*) page$""") {
```

Comment:
Shall we keep those consistent?
`"""^(?:I )?click radio option (.*) on the 'Are you sure you want to remove goods item number (.*)\?' page$"""`